### DISCUSSION TITLE
あなたの出身地はどこ？- WHERE IS YOUR BIRTHPLACE?

### DISCUSSION THREAD

**Discussion by yoriokahiroto** [NAIST-SE/AttractingContributors#469] [General]
TELL ME YOUR BIRTHPLACE.
<あなたの出身はどこですか？>
IT MIGHT BE GOOD OPPORTUNITY TO INTRODUCE YOURSELF TO OTHERS.
<みんなに自分のことを知ってもらえる機会になるかも？>

是非、答えてくださいね！！！
<I'm looking forward you joining it!!!>

fork from here please
https://github.com/yoriokahiroto/YourBirthPlace

**Comment by aru1702** [comment]
こんにちは！
インドネシアから僕はジャカルタ市出身です。
よろしくねー

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
